portlet-tck_3.0/pom.xml

```diff
@@ -216,12 +216,18 @@
             <version>4.11</version>
             <scope>test</scope>
          </dependency>
-         <dependency>
-            <groupId>org.seleniumhq.selenium</groupId>
-            <artifactId>selenium-java</artifactId>
-            <version>2.53.0</version>
-            <scope>test</scope>
-         </dependency>
+         <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-api</artifactId>
+            <version>3.11.0</version>
+            <scope>compile</scope>
+         </dependency>
+         <dependency>
+            <groupId>org.seleniumhq.selenium</groupId>
+            <artifactId>selenium-java</artifactId>
+            <version>3.11.0</version>
+            <scope>compile</scope>
```

Comment:
@vsingleton Same question here about the change in scope.

Reply:
This one I was able to switch to >test<